Makefile

```diff
@@ -0,0 +1,31 @@
+# this is a file to create build directories here
```

Comment:
We already have a place for build directories: `M2/BUILD`.  And instead of letting the user make them, just make them and populate each one with appropriate files for the user to use.  The names could be `M2/BUILDS/cmake/debug`, etc.

Reply:
@mikestillman do you need this file?

Reply:
@mahrud Please remove that top level Makefile.  I no longer use it.